pydantic/_migration.py

```diff
@@ -12,6 +12,7 @@
     'pydantic.utils:to_lower_camel': 'pydantic.alias_generators:to_camel',
     'pydantic:PyObject': 'pydantic.types:ImportString',
     'pydantic.types:PyObject': 'pydantic.types:ImportString',
+    'pydantic.generics.GenericModel': 'pydantic.BaseModel',
```

Comment:
I think `GenericModel` should either on `REMOVED_IN_V2` or it should have a special condition like we have on `BaseSettings` on the `getattr_migration`. 🤔

Reply:
Not sure what you mean, can you just push your suggestion?